docs/user/faq.md

```diff
@@ -94,6 +94,12 @@ x86_64-apple-darwin13.4.0-clang: No such file or directory
 
 are a telltale sign that you are lacking compilers.
 
+Depending on the specific tools that you use for building C/C++ code, you may need some additional tools, for example:
+
+```
+c-compiler cxx-compiler cmake pkg-config make ninja meson
```

Comment:
```suggestion
cxx-compiler cmake pkg-config make ninja meson
```

It seems that asking for `c-compiler` is fully redundant as `cxx-compiler` installs `c-compiler`

```console
$ mkdir compiler-diff && cd compiler-diff
$ pixi init
$ pixi add cxx-compiler
$ pixi list &> cxx-install.txt
$ pixi add c-compiler
$ pixi list &> c-install.txt
$ diff cxx-install.txt c-install.txt  # no output
$ grep c-compiler cxx-install.txt 
c-compiler                1.8.0    h2b85faf_1    5.9 KiB    conda  c-compiler-1.8.0-h2b85faf_1.conda
```

Reply:
Good catch! Indeed that is the case, see https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Fcxx-compiler-1.8.0-h1a2810e_1.conda . Probably I had experience with older `cxx-compiler` builds without these dependencies, or the fact that `compiler` directive in conda recipes do not have these dependency.

Reply:
:+1: I only noticed this by chance today when I ran a `pixi list` on a test env and saw `c-compiler` when my `pixi.toml` had only `cxx-compiler`. I remembered this PR then and thought I'd investigate a bit. :)